pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -51,6 +51,17 @@ var (
 // the pod will be moved from unschedulableQ to activeQ.
 const unschedulableQTimeInterval = 60 * time.Second
 
+const (
+	// DefaultPodInitialBackoffDurationSeconds is the default value for the initial backoff duration
+	// for unschedulable pods. To change the default podInitialBackoffDurationSeconds used by the
+	// scheduler, update the ComponentConfig value in defaults.go
+	DefaultPodInitialBackoffDurationSeconds int64 = 1
```

Comment:
> I would suggest that we name this constant different. In the context of this package, it makes total sense if it's just `DefaultInitialBackoffDuration` and it's of type `time.Duration`.

I prefer to keep the name as it is and use time.Duration as its type

Reply:
> +1 to Aldo's suggestion in shorting the name and using type Duration. We don't need to "fully qualify" the variable name here since the context is clear. We should also make it local to the package and perhaps move both `queueClosed` and `unschedulableQTimeInterval` also inside the `const (` close.

If we make it a local var, we can't reference it in the scheduler package and go back to the first implementation with duplicate hard-coded constants.